docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -6,9 +6,9 @@
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using encrypted smart contracts that run on the {IC} blockchain.
+The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different pseudonym is created for each application you use, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using smart contracts that run on the {IC} blockchain.
 
-Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you create authentication profiles using a identity anchor and the authentication methods you choose such as facial recognition from a smartphone, your computer unlock password, or a security key. 
+Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you create authentication profiles using an identity anchor and the authentication methods you choose such as facial recognition from a smartphone, your computer unlock password, or a security key.
```

Comment:
I think we should not say "your Internet Identity" – the user "only" has an identity anchor. Or we could say "Internet Identity does not require ..."

Reply:
Agreed, done.

Reply:
Please also remove "your" from " your Internet Identity"
There is no single Internet Identity.